### PR TITLE
Support gitlab!

### PR DESCRIPTION
**Pull request by laixintao**
Hi @ruanyl , we use gitlab now

I add some changes that support gitlab url, basically like Github's url scheme.

### CODE REVIEW

File: plugin/vim-gh-line.vim [hunk 1]
```diff
@@ -77,6 +77,10 @@ func! s:gh_line() range
           let lineRange = 'L' . a:firstline . "-L" . a:lastline
       endif
       let url = origin . blob . commit . relative . '#' . fileName . lineRange
+     elseif exists("g:gh_gitlab_domain")
```
Comment (ruanyl) [inline plugin/vim-gh-line.vim]: Would it be better to check `g:gh_gitlab_domain` against `origin`?

Comment (laixintao) [reply plugin/vim-gh-line.vim]: Origin maybe self setted, like deploy a gitlab server and set hostname is 'code-base.com'

Comment (laixintao) [reply plugin/vim-gh-line.vim]: Maybe check `exists("g:gh_gitlab_domain")` and `match(origin, "g:gh_gitlab_domain")` would be better?

Comment (ruanyl) [reply plugin/vim-gh-line.vim]: @laixintao that's what I mean 👍

**Comment by laixintao** [comment]
Done!

**Comment by ruanyl** [comment]
Thanks, it looks great to me 👍

**Pull request merged**